### ISSUE TITLE
Get-ValueFromALTestRunnerConfig : Cannot process argument transformation on parameter 'ConfigPath'. Cannot convert value to type System.String.

### ISSUE THREAD

**Issue by JoelJame5** [jimmymcp/al-test-runner#36]
The [v0.5.5 changes](https://github.com/jimmymcp/al-test-runner/commit/a521bf2c8b87ec172af04a7a84faa3e030ad0855) now causes following error when running the tests.  

```
Get-ValueFromALTestRunnerConfig : Cannot process argument transformation on parameter 'ConfigPath'. Cannot convert value to type System.String.
At C:\Users\joel.james\.vscode\extensions\jamespearson.al-test-runner-0.5.5\PowerShell\Get-DockerHostIsRemote.ps1:2 char:19      
+ ...   $dockerHost = Get-ValueFromALTestRunnerConfig -KeyName 'dockerHost'
+                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Get-ValueFromALTestRunnerConfig], ParameterBindingArgumentTransformationException
    + FullyQualifiedErrorId : ParameterArgumentTransformationError,Get-ValueFromALTestRunnerConfig
```

I made the following change locally to Get-ALTestRunnerConfigPath to get around the issue.
![image](https://user-images.githubusercontent.com/27202678/118643757-e0f47900-b7d4-11eb-92d8-4b505793f0dd.png)

PS: Loved your latest video!! Proper youtuber now, keep 'em coming! :)

**Comment by jimmymcp** [comment]
Hi mate,

Sorry, my bad - I didn't test this against single-root workspaces. Should be fixed in the new version.

Hope that you're all well @ Clever Dynamics HQ 👍

**Comment by jimmymcp** [comment]
I published 0.5.6 over lunch. Should be able to get it from the marketplace now.

**Comment by lvanvugt** [comment]
LOL, just now it showed **Reload required**.

Running right now ... and ... **WORKING**.
